Posh-ACME.Deploy/Private/Confirm-CertInstall.ps1

```diff
@@ -1,8 +1,10 @@
 function Confirm-CertInstall {
     param(
         [Parameter(Position=0)]
+        [AllowEmptyString()]

```

Comment:
Is adding an explicit `AllowEmptyString` doing something I'm missing? Since neither param is mandatory, they already allow empty strings by default, right?

Reply:
That is what I thought, AND it was what I observed on Windows Server 2019.  On Windows Server 2016 there were complaints until I added that explicit language.

Reply:
Oh weird. I mean, it obviously doesn't hurt. Just seemed odd.